### ISSUE TITLE
Is this repo still active?

### ISSUE THREAD

**Issue by TkDodo** [remeda/remeda#87]
Should we be worried that there does not seem to be any traction on that repo for a couple of months now? Is this repo still maintained? What can we do to get the open PRs merged for example ... ?

**Comment by lstkz** [comment]
Yes! Sorry for the late reply, I am working on it :wink:

**Comment by TkDodo** [comment]
Nice, thanks a lot 👍 .

**Comment by Bessonov** [comment]
There is no movement since 1,5 years. What are the plans for the library? @lstkz

**Comment by TkDodo** [comment]
Hi 👋 

I have just been added as a maintainer of this library, and I plan on being active here :)
We will do our best to make sure that this library stays actively maintained 🚀

**Comment by Bessonov** [comment]
Cool, thank you for the light-fast response! That makes hope for this great library. Which priorities do you see or will take?

**Comment by TkDodo** [comment]
I'm going to talk to @lstkz about it. My personal prio would be to get a v1 out. I don't think we plan on breaking changes - most open issues and PRs are just about adding features as far as I can see. Happy to discuss it with you as well obviously. Having Github discussions enabled for these things would be nice, too :)

**Comment by Bessonov** [comment]
From my point of view it's important to get a review on semantic and usage of some functions. I don't use this library yet, because I've discovered it yesterday. Therefore I can't say that the semantic matches my(!) expectations without reviewing everything. But I found that the semantic of `merge` [doesn't makes me happy](https://github.com/remeda/remeda/pull/69#issuecomment-846888214).

Another topic to discuss is https://github.com/remeda/remeda/issues/37 .
